cpp/src/IceStorm/Makefile.mk

```diff
@@ -4,8 +4,6 @@
 
 $(project)_libraries            := IceStormService
 $(project)_programs             := icestormadmin icestormdb
-$(project)_sliceflags           := -Isrc --include-dir IceStorm
```

Comment:
Note that this PR does not add (or remove) any include directory from the C++ build.